linera-views/src/batch.rs

```diff
@@ -108,6 +108,21 @@ impl UnorderedBatch {
             insertions,
         })
     }
+
+    /// For convenience in some tests, it it good to do this suboptimization
+    pub fn as_batch(self) -> Batch {
```

Comment:
Functions that consume and convert `self` are usually called `into_`, not `as_`.

```suggestion
    pub fn into_batch(self) -> Batch {
```

Reply:
I actually removed that function `as_batch` since it is not really needed.